README.md

```diff
@@ -1 +1,81 @@
-# th2-conn-ws-client
\ No newline at end of file
+# WebSocket Client
+
+This microservice allows sending and receiving of messages via WebSocket protocol
+
+## Configuration
+
+Main configuration is done via setting following properties in a custom configuration:
+
++ **uri** - connection URI
++ **sessionAlias** - session alias for incoming/outgoing TH2 messages (e.g. `ws_api`)
++ **handlerSettings** - WebSocket event handler settings
+
+### Event handler configuration
+
+Event handler can be configured via setting following properties in the `handlerSettings` block of the main configuration
+
++ **pingInterval** - interval for sending ping-messages in ms (`30000` by default)
+
+### Configuration example
+
+```yaml
+uri: wss://echo.websocket.org
+sessionAlias: api_session
+handlerSettings:
+  pingInterval: 30000
+```
+
+### MQ pins
+
+* input queue with `subscribe` and `send` attributes for outgoing messages
+* output queue with `publish` and `first` (for incoming messages) and `second` (for outgoing messages) attributes
+
+## Inputs/outputs
+
+This section describes messages received and by produced by the service
```

Comment:
This section describes **the** messages **received and** produced by the service